This resolves some "declaration of C function conflicts with previous
declaration" errors during compilation.

--- a/src/FFmpeg.h	2015-02-21 00:33:33.853857529 +0100
+++ b/src/FFmpeg.h	2015-02-21 00:35:09.626497205 +0100
@@ -688,7 +688,7 @@
    FFMPEG_FUNCTION_WITH_RETURN(
       AVOutputFormat*,
       av_oformat_next,
-      (AVOutputFormat *f),
+      (const AVOutputFormat *f),
       (f)
    );
    FFMPEG_FUNCTION_WITH_RETURN(
@@ -755,7 +755,7 @@
    FFMPEG_FUNCTION_WITH_RETURN(
       int,
       av_fifo_size,
-      (AVFifoBuffer *f),
+      (const AVFifoBuffer *f),
       (f)
    );
    FFMPEG_FUNCTION_WITH_RETURN(
@@ -801,7 +801,7 @@
    FFMPEG_FUNCTION_WITH_RETURN(
       AVDictionaryEntry *,
       av_dict_get,
-      (AVDictionary *m, const char *key, const AVDictionaryEntry *prev, int flags),
+      (const AVDictionary *m, const char *key, const AVDictionaryEntry *prev, int flags),
       (m, key, prev, flags)
    );
    FFMPEG_FUNCTION_WITH_RETURN(
